module/VuFind/src/VuFind/ILS/Driver/Alma.php

```diff
@@ -1092,7 +1108,9 @@ public function renewMyItems($renewDetails)
      */
     public function getStatus($id)
     {
-        return $this->getHolding($id);
+        $idList = [$id];
```

Comment:
Since the return format of getHolding has changed and the return format of getStatus has not, proxying getHolding is no longer a good idea here. Actually, I'm pretty sure it was never a good idea, so I've refactored this to be a wrapper around getStatuses (as also done in the Horizon driver, for example).